test/e2e/network/netpol/network_policy_api.go

```diff
@@ -209,4 +210,76 @@ var _ = common.SIGDescribe("Netpol API", func() {
 		framework.ExpectNoError(err)
 		framework.ExpectEqual(len(nps.Items), 0, "filtered list should have 0 items")
 	})
+
+	/*
+		Release: v1.21
+		Testname: NetworkPolicy support EndPort Field
+		Description:
```

Comment:
@knabben  is it enough to cover our api validation?

Reply:
looks like good coverage.

@rikatz any other missing case?

Reply:
@jorn you can resolve this comment, we add more tests later if necessary.